src/RestrictedPython/transformer.py

```diff
@@ -627,9 +627,12 @@ def visit_Expr(self, node):
 
     def visit_UnaryOp(self, node):
         """
-
+        UnaryOp (Unary Operations) is the overall element for:
+        * Not --> which should be allowed
+        * UAdd
+        * USub
         """
-        self.not_allowed(node)
+        return self.node_contents_visit(node)
```

Comment:
It was only forbidden because there was no test for this method. 
Did you write a test for it?